cmd/bosun/expr/funcs.go

```diff
@@ -104,24 +107,35 @@ var TSDB = map[string]parse.Func{
 		parse.TypeSeries,
 		tagQuery,
 		Band,
+		nil,
 	},
 	"change": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeNumber,
 		tagQuery,
 		Change,
+		nil,
 	},
 	"count": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeScalar,
 		nil,
 		Count,
+		nil,
```

Comment:
Yeah, that would cause a ton more changes, but may make things more readable throughout. Your call.